src/nft/components/details/detailsV2/LandingPage.tsx

```diff
@@ -40,43 +44,40 @@ const InfoContainer = styled(ColumnCenter)`
   }
 `
 
+const StyledHeadlineText = styled(ThemedText.HeadlineSmall)``
+const StyledSubheaderText = styled(ThemedText.SubHeaderSmall)`
+  line-height: 20px;
+`
+
 const InfoDetailsContainer = styled(Column)`
   gap: 4px;
   align-items: center;
+
+  @media screen and (min-width: ${BREAKPOINTS.sm}px) {
+    ${StyledHeadlineText} {
```

Comment:
This is actually going off Tina's comment on an older merged PR. The problem is we would need !important on these styles so instead of doing that she suggested to do this.

Reply:
oh interesting. something feels funky here to me - maybe it's that we need a separate `ResponsiveText` or we just shouldn't use the default text here since it doesn't have the same behavior as the rest of the `HeadlineSmall` components

Reply:
Yeah I agree I think it makes more sense to just not use the ThemedText here